recipes/modern-cpp-kafka/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class ModernCppKafkaConan(ConanFile):
+    name = "modern-cpp-kafka"
+    description = "A C++ API for Kafka clients (i.e. KafkaProducer, KafkaConsumer, AdminClient)"
+    license = "Apache License 2.0"
+    topics = ("conan", "modern-cpp-kafka", "kafka", "librdkafka", "kafkaproducer", "kafkaconsumer")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/morganstanley/modern-cpp-kafka"
+    exports_sources = "include/*"
```

Comment:
```suggestion
    settings = "arch", "build_type", "compiler", "os"
```
There are not header to export here. Please, read https://docs.conan.io/en/latest/creating_packages/package_repo.html#exporting-the-sources-with-the-recipe-exports-sources to understand exports_sources usage.

Keep full settings always. You can customize the package ID on `package_id` method.

Reply:
Fixed in the latest commit.